src/app/(proper_react)/redesign/(public)/SignUpForm.tsx

```diff
@@ -40,7 +48,12 @@ export const SignUpForm = (props: Props) => {
         className={props.isHero ? styles.isHero : ""}
         name={emailInputId}
         id={emailInputId}
-        onChange={(e) => setEmailInput(e.target.value)}
+        onChange={(e) => {
+          setEmailInput(e.target.value);
+          record("field", "focus", {
```

Comment:
is "focus" the right event/thing here?
doesn't `onChange` trigger this event for every keystroke?

Reply:
Hm yeah I think the name is confusing... it sounds like it would only be for `onFocus`, I don't think it matters too much.

GA4 calls any interaction with the form `form_start` and it also has a `form_submit` event https://support.google.com/analytics/answer/9216061?hl=en, we should consider changing the Glean name to either be the same or maybe something like `form_interaction`.

I don't think we should do that in this PR though, @codemist would you mind using it the way you are for now and file a ticket in the backlog to change the name of this later?

Reply:
@codemist I think the idea is to just generate an event if the user interacts at all with the form, if this does generate a new event on every keypress that's probably way more than we need. 

Is it possible to have this fire when the form is focused, vs. the input field? Do we use auto focus at all now?